modules/swagger-codegen/src/main/resources/pkmst/readme.mustache

```diff
@@ -142,3 +142,14 @@ http://localhost:8008/swagger-ui.html
 If all the configurations have been enabled(depending on the port) below are some of the URls to access:
 eureka: http://localhost:8080
 zipkin: http://localhost:9411
+
+Please visit below links for more information on PKMST:
+
+Getting started
+https://pkmst-getting-started.mybluemix.net
```

Comment:
I got the following when accessing https://pkmst-getting-started.mybluemix.net:
```
404 Not Found: Requested route ('pkmst-getting-started.mybluemix.net') does not exist.
```

Reply:
@wing328 It was under maintenence. now it's up and we are currently updating the content according to swagger codegen.

Reply:
Good. I'll send out a tweet to promote the new generator.